### PR TITLE
docs: Install Appsmith on Azure Container Instance

### PR DESCRIPTION
**Pull request by sum35h**
First draft to install Appsmith on Azure Container Instance with Azure File Share mount for data persistance.

### CODE REVIEW

**Comment by jnikhila** [comment]
@sum35h - Does this need internal review or is it ready for doc team?

**Comment by jnikhila** [comment]
@sum35h - Added a doc ticket to track this - #772

File: website/docs/getting-started/setup/installation-guides/azure-aci.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+sidebar_position: 4
+---
+
+# Azure Container Instance
+Azure Container Instances (ACI) is a fast and simple way to run containers in the cloud.
+In this document, we will guide you through the process of launching an ACI instance and running an Appsmith container on it. The data will be persisted to an Azure Storage Account File Share 
+note: Azure only supports CIFS file shares and NFS file shares are not yet supported.
+
+### Prequisite
+Before you can launch an ACI instance, you need to have an Azure subscription and have the Azure CLI installed on your machine. If you do not have an Azure subscription, you can sign up for a free trial at **[https://azure.com/free](https://azure.com/free)**.
```
Comment (danciaclara) [inline website/docs/getting-started/setup/installation-guides/azure-aci.md:12]: Can we list the pre-requisites in bulleted format and in the order they need to be installed assuming the user maybe doing it for the first time? 

So something like

Before launching an ACI instance, you need to ensure that the following prerequisites are met.

- [Azure subscription](https://azure.com/free)
- Azure CLI 
- [Create an account on AWS]((https://aws.amazon.com/premiumsupport/knowledge-center/create-and-activate-aws-account/)) if you don't already have one.

File: website/docs/getting-started/setup/installation-guides/azure-aci.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+If you already have an Amazon Web Services account, you may skip this step.
+
+Please follow the steps [detailed here](https://aws.amazon.com/premiumsupport/knowledge-center/create-and-activate-aws-account/) in order to create an account on AWS.
+
+
+### Set Basic variable details
```
Comment (danciaclara) [inline website/docs/getting-started/setup/installation-guides/azure-aci.md:19]:
```suggestion
### Configure variables
```

File: website/docs/getting-started/setup/installation-guides/azure-aci.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+###Create a storage account (optional)
+`az storage account create --resource-group $resourceGroupName --name $storageAccountName --location $aciLocation --sku Standard_LRS`
+### Get the storage account key
+`storageAccountKey=$(az storage account keys list --resource-group $resourceGroupName --account-name $storageAccountName --query "[0].value"  --output tsv)`
+
+### Create a file share
+`az storage share create --name $fileShareName --account-name $storageAccountName --account-key $storageAccountKey`
+
+### Create an Appsmith ACI with the storage account and file share mounted
```
Comment (danciaclara) [inline website/docs/getting-started/setup/installation-guides/azure-aci.md:40]:
```suggestion
### Create Appsmith ACI
```

mention the the storage account and file share mounted in the intro of the section

File: website/docs/getting-started/setup/installation-guides/azure-aci.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+ --image appsmith/appsmith-ce \
+ --ports 80 443 \
+ --environment-variables 'APPSMITH_LICENSE_KEY'='<Your Appsmith License>' \
+ --azure-file-volume-account-name $storageAccountName \
+ --azure-file-volume-account-key $storageAccountKey \
+ --azure-file-volume-share-name $fileShareName \
+ --azure-file-volume-mount-path "/appsmith-stacks/"`
+
+
+## Updating to the latest Appsmith release
```
Comment (danciaclara) [inline website/docs/getting-started/setup/installation-guides/azure-aci.md:64]:
```suggestion
## Updating Appsmith
```

**Review by danciaclara — COMMENTED** [review]
Just leaving comments cc: @jnikhila

File: website/docs/getting-started/setup/installation-guides/azure-aci.md [hunk 5]
```diff
@@ -0,0 +1,130 @@
...
+:::note 
+Azure only supports CIFS file shares and doesn't support NFS file shares.
+:::
+
+### Prerequisites​
+
+Before launching an ACI instance, you need to have an Azure subscription and have the Azure CLI installed on your machine. 
+
+* [Azure Subscription](https://azure.com/free) - If you don't have an Azure subscription, you can sign up for a free trial
+* Azure CLI
```
Comment (harshilp24) [inline website/docs/getting-started/setup/installation-guides/azure-aci.md:21]: @sum35h can we link Azure CLI to https://learn.microsoft.com/en-us/cli/azure/  ?

Comment (sum35h) [reply website/docs/getting-started/setup/installation-guides/azure-aci.md:21]: Sure, lets add it.

Comment (sum35h) [reply website/docs/getting-started/setup/installation-guides/azure-aci.md]: Sure, I'm adding it.

**Review by harshilp24 — APPROVED** [review]

**Review by danciaclara — APPROVED** [review]

**Pull request merged**